Updated collect logs article (#227)
AdguardTeam/KnowledgeBase: docs/adguard-for-android/solving-problems/log.md

:::

In this article, we'll guide you through the process of collecting debug logs, an essential troubleshooting step for solving complex issues that may potentially arise. Debug logs provide detailed insight into the inner workings of AdGuard for Android. If the AdGuard support team askes you to provide debug logs, simply follow these instructions.

### Collecting debug log

To collect **debug** log and send it to us you need to perform following steps:

1. Go to *Settings* → *General* → *Advanced*.
2. Tap *Logging level* and set it to *Debug*.
3. Reproduce the problem and try to remember the exact time it occurred.
4. Wait a while, then return to *Settings* and choose the *Support* tab. Tap *Report a bug* and complete the required fields. Don't forget to check the *Send app logs and system info* checkbox. Finally tap *Send*.

> or to act more effective

Reviewer: ?

> 4. Wait a while, then return to the *Advanced* screen and export logs via *Export logs and system info*. Then report a bug on GitHub by following these [instructions](../guides/repor-bugs.md).

Reviewer: два раза четвёртый пункт, а между ними текст — непонятно

> 5. After creating an issue on GitHub send the log file to devteam@adguard.com. Specify the time of the error and attach a link to your issue or its number (it appears as #number next to the title).

Reviewer: ```suggest
5. After creating an issue on GitHub, send the log file to devteam@adguard.com. Include the time of the bug and attach a link to your issue or its number (it appears as #number next to the title).
Alternatively, you can upload the log file to Google Drive and send it to devteam@adguard.com. Add the file link to your GitHub issue
```

3. Reproduce the problem and try to remember the exact time it occurred.
4. Wait a while, then return to *Settings* and choose the *Support* tab. Tap *Report a bug* and complete the required fields. Don't forget to check the *Send app logs and system info* checkbox. Finally tap *Send*.

> If you're interested in following the resolution of your issue and engaging in a dialogue with the developers, we recommend taking the following steps after completing the initial three:

Reviewer: ```suggestion
If you're interested in following the resolution of your issue and engaging in a dialogue with the developers, we recommend that you take the following steps after completing the first three:
```
